build/testdata/003.golden

```diff
@@ -3,6 +3,24 @@ numbers = [
     11,
     123.456,
     123.,
-    .456,
+    0.456,
     1.23e45,
+    -1,
+    +1,
+    0.0,
+    -0.0,
+    1.0,
+    -1.0,
+    +1.0,
+    1e6,
+    -1e6,
+    -1.23e-45,
+    3.539537889086625e24,
+    3.539537889086625e24,
+    3.539537889086625e24000,
+    3.539537889086625e24000,
+    3539537889086624823140625,
+    0x123,
+    0xE45,
```

Comment:
Shouldn't this `E` also be lowercased?

Reply:
Or this is invalid and just left as is?

Reply:
This is not a float, but a hex number. We could discuss whether hex literals should be canonicalized in some way, but I would prefer to handle that in a separate PR.